lib/aweplug/books/google_books.rb

```diff
@@ -85,7 +85,7 @@ def get data
           unless required_keys.all? {|key| book['volumeInfo'].key? key}
             isbn = isbn_13(book) || data['isbn']
             puts "book: #{isbn} missing required attributes: #{required_keys - book['volumeInfo'].keys}"
-            return nil 
```

Comment:
don't combine whitespace changes with code changes. It makes it hard to detect the actual changes.